src/rnp/rnp.cpp

```diff
@@ -281,6 +281,7 @@ setcmd(rnp_cfg_t *cfg, int cmd, const char *arg)
         break;
     case CMD_SYM_ENCRYPT:
         rnp_cfg_setbool(cfg, CFG_ENCRYPT_SK, true);
+        rnp_cfg_setint(cfg, CFG_KEYSTORE_DISABLED, 1);
```

Comment:
This will not be correct since user may use public-key and symmetric encryption simultaneously, or want to symmetrically encrypt and sign file.

Reply:
Also, together with such changes, tests should be added which makes sure that issue is actually fixed (and do not have regression later on).

Reply:
So what is the right "... one-line fix (via CFG_KEYSTORE_DISABLED) ..." as @dewyatt required above?

Reply:
@joke325 Your fix is probably what I would have done, as it fixes the issue I referred to in my comment, but as @ni4 mentioned it probably introduces a separate issue, so maybe a few more lines will be required. :)